python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -20,48 +20,87 @@
+{%                 set vlans = [] %}
+{%                 if port_channel_interface.switchport.access_vlan is arista.avd.defined %}
+{%                     do vlans.append(port_channel_interface.switchport.access_vlan) %}
+{%                 endif %}
+{%                 if port_channel_interface.switchport.trunk.allowed_vlan is arista.avd.defined %}
+{%                     set allowed_vlans = port_channel_interface.switchport.trunk.allowed_vlan | string | arista.avd.range_expand %}
+{%                     for allowed_vlan in allowed_vlans %}
+{%                         do vlans.append(allowed_vlan | int) %}
+{%                     endfor %}
+{%                 endif %}
+{%                 set vlans = vlans | arista.avd.list_compress | arista.avd.default('-') %}
+{%                 if port_channel_interface.switchport.trunk.native_vlan_tag is arista.avd.defined(true) %}
+{%                     set native_vlan = "tag" %}
+{%                 else %}
+{%                     set native_vlan = port_channel_interface.switchport.trunk.native_vlan | arista.avd.default('-') %}
+{%                 endif %}
+{%                 set channel_group = ethernet_interface.channel_group.id %}
+{%                 set trunk_groups = port_channel_interface.switchport.trunk.groups | arista.avd.default('-') %}
+| {{ ethernet_interface.name }} | {{ description }} | *{{ mode }} | *{{ vlans }} | *{{ native_vlan }} | *{{ trunk_groups }} | {{ channel_group }} |
+{%             elif port_channel_interface.type is arista.avd.defined('switched') %}
+{%                 set description = ethernet_interface.description | arista.avd.default("-") %}
+{%                 set mode = port_channel_interface.mode | arista.avd.default('-') %}
 {%                 set vlans = port_channel_interface.vlans | arista.avd.default('-') %}
 {%                 if  port_channel_interface.native_vlan_tag is arista.avd.defined(true) %}
 {%                     set native_vlan = "tag" %}
 {%                 else %}
 {%                     set native_vlan = port_channel_interface.native_vlan | arista.avd.default('-') %}
 {%                 endif %}
 {%                 set channel_group = ethernet_interface.channel_group.id %}
-{%                 if port_channel_interface.trunk_groups is arista.avd.defined %}
-{%                     set l2 = namespace() %}
-{%                     set l2.trunk_groups = [] %}
-{%                     for trunk_group in port_channel_interface.trunk_groups | arista.avd.natural_sort %}
-{%                         do l2.trunk_groups.append(trunk_group) %}
+{%                 set trunk_groups = port_channel_interface.trunk_groups | arista.avd.default('-') %}
+| {{ ethernet_interface.name }} | {{ description }} | *{{ mode }} | *{{ vlans }} | *{{ native_vlan }} | *{{ trunk_groups }} | {{ channel_group }} |
+{%             endif %}
+{%         else %}
+{%             if (ethernet_interface.switchport.mode is arista.avd.defined or
```

Comment:
Please add some `{# comment #}` around this huge if, to explain why we have it. Also a TODO for 6.0.0 to remove it once the old keys are removed.

Reply:
Fixed